2005-11-01  Kazuhiro Inaoka <inaoka.kazuhiro@renesas.com>

	* Makeconfig (libc.so-gnulib): Added -lgcc_s$(libgcc_s_suffix)
	to avoid undefined _Unwind_Find_FDE.

Index: Makeconfig
===================================================================
RCS file: /cvs/glibc/libc/Makeconfig,v
retrieving revision 1.302.2.1
diff -u -p -r1.302.2.1 Makeconfig
--- Makeconfig	16 Feb 2005 10:50:19 -0000	1.302.2.1
+++ Makeconfig	9 May 2006 05:46:37 -0000
@@ -509,7 +509,7 @@ else
 endif
 gnulib := -lgcc $(libgcc_eh)
 static-gnulib := -lgcc -lgcc_eh $(libunwind)
-libc.so-gnulib := -lgcc
+libc.so-gnulib := -lgcc -lgcc_s$(libgcc_s_suffix)
 endif
 ifeq ($(elf),yes)
 +preinit = $(addprefix $(csu-objpfx),crti.o)
